packages/ts/react-i18n/src/index.ts

```diff
@@ -182,6 +191,32 @@ export class I18n {
     });
   }
 
+  /**
+   * Reloads all translations for the current language. This method should only
+   * be used for HMR in development mode.
+   */
+  private async reloadTranslations() {
+    const currentLanguage = this.#language.value;
+    if (!currentLanguage) {
+      return;
+    }
+
+    let translationsResult: TranslationsResult;
+    try {
+      translationsResult = await this.#backend.loadTranslations(currentLanguage);
```

Comment:
Should the cache for other languages be cleared here also? If I change german translations and are looking at English ones, I want to see the updated german strings when switching the language back to german

Reply:
There is no cache, when you change the language it always loads all translations that have been loaded so far again.